### ISSUE TITLE
pymzn doesnt work with minizinc 2.5.0

### ISSUE THREAD

**Issue by vrvarma** [paolodragone/pymzn#48]
Was trying to run pymzn on minizinc 2.5.0.  To reproduce, we can use the default example.  Looks like the new version of minizinc is adding extra information in the output that pymzn is not able to parse the output properly.

`pymzn minizinc knapsack.mzn knapsack.dzn --data '{"capacity": 20}'`

```

Traceback (most recent call last):
  File "/usr/local/bin/pymzn", line 8, in <module>
    sys.exit(main())
  File "/usr/local/lib/python3.8/dist-packages/pymzn/__init__.py", line 222, in main
    args['func'](**args)
  File "/usr/local/lib/python3.8/dist-packages/pymzn/__init__.py", line 43, in _minizinc
    solns = minizinc(
  File "/usr/local/lib/python3.8/dist-packages/pymzn/mzn/minizinc.py", line 687, in minizinc
    solns = parser.parse(proc)
  File "/usr/local/lib/python3.8/dist-packages/pymzn/mzn/output.py", line 245, in parse
    self._collect(proc, solns)
  File "/usr/local/lib/python3.8/dist-packages/pymzn/mzn/output.py", line 249, in _collect
    for soln in self._parse(proc):
  File "/usr/local/lib/python3.8/dist-packages/pymzn/mzn/output.py", line 263, in _parse
    soln = parse_lines.send(line)
  File "/usr/local/lib/python3.8/dist-packages/pymzn/mzn/output.py", line 279, in _parse_lines
    soln = dzn2dict(
  File "/usr/local/lib/python3.8/dist-packages/pymzn/dzn/parse.py", line 552, in dzn2dict
    raise ValueError(
ValueError: Unsupported parsing for statement:
'MiniZinc to FlatZinc converter, version 2.5.0, build 198486215\nCopyright (C) 2014-2020 Monash University, NICTA, Data61\nx = {3,5}'
```

**Comment by paolodragone** [comment]
Yes, apparently they've changed the format of the output of a Minizinc call. So yes, in the short term it's probably better to just downgrade to the 2.4.* version. If there is a strong reason to use 2.5.0 please let me know, for now I'll wait a bit for 2.5.* to become a bit more stable before adapting PyMzn.

**Comment by vrvarma** [comment]
There was bug in libminizinc "Print verbose version information to stderr instead of stdout."  This issue has been fixed in 2.5.1.  
There is no issue with pymzn..  I am going to close this defect.

**Comment by paolodragone** [comment]
Thanks @vrvarma!
